src/IROperator.h

```diff
@@ -188,6 +188,10 @@ struct BufferBuilder {
     EXPORT Expr build() const;
 };
 
+/** If e is a ramp expression with stride, default 1, return the base,
+ * otherwise undefined. */
+Expr strided_ramp_base(Expr e, int stride = 1);
```

Comment:
The code that calls this is pretty confusing when it uses the default argument of 1. I couldn't figure out what was going on until I found this declaration.